wpimath/src/main/native/include/frc/kinematics/proto/ChassisSpeedsProto.h

```diff
@@ -4,6 +4,8 @@
 
 #pragma once
 
+#include <google/protobuf/arena.h>
+#include <google/protobuf/message.h>
```

Comment:
We do not want to include these here.  We do a lot of work in Protobuf.h etc to make sure that it's not necessary to include the protobuf headers.  It should not be necessary and substantially increases compile time.